templates/vue/repo-search.tmpl

```diff
@@ -0,0 +1,81 @@
+<script type="text/x-template" id="repo-search-template">
```

Comment:
Where should I put it then? It's here for 2 (temporary) reasons:

1. There are part rendered by server
2. You can include it on any page as template for component (can't be included in js for reason 1)